collect_app/src/androidTest/java/org/odk/collect/android/AllWidgetsFormTest.java

```diff
@@ -846,189 +555,72 @@ public void testImageSelectOne() {
-            }
-        };
-    }
-
-    public static ViewAction setNumberPickerValue(final int value) {
-        return new ViewAction() {
-            @Override
-            public void perform(UiController uiController, View view) {
-                NumberPicker numberPickerDialog = (NumberPicker) view;
-                numberPickerDialog.setValue(value);
-            }
-
-            @Override
-            public String getDescription() {
-                return "Set a value on a Number Picker";
-            }
-
-            @Override
-            public Matcher<View> getConstraints() {
-                return ViewMatchers.isAssignableFrom(NumberPicker.class);
-            }
-        };
-    }
-
-    private ViewInteraction onVisibleSeekBar() {
-        return onView(withId(R.id.seek_bar));
-    }
-
-    private ViewInteraction onVisibleEditText() {
-        return onView(withClassName(endsWith("EditText")));
-    }
-
-    private ViewInteraction onVisibleCustomEditText() {
-        return onView(withClassName(endsWith("CustomEditText")));
-    }
-
-    private ViewInteraction onVisibleCheckBox() {
-        return onView(withClassName(endsWith("CheckBox")));
-    }
-
-    private ViewInteraction onVisibleNumberPickerDialog() {
-        return onView(withClassName(endsWith("NumberPicker")));
-    }
-
-    // private void openWidget(String name) {
-    //     openWidgetList();
-    //     onView(withText(name)).perform(click());
-    // }
-
-    private void openWidgetList() {
-        onView(withId(R.id.menu_goto)).perform(click());
-    }
-
-    // private void saveForm() {
-    //    onView(withId(R.id.menu_save)).perform(click());
-    // }
-
-    private String randomString() {
-        return RandomString.make();
-    }
-
-    private int randomInt() {
-        return Math.abs(random.nextInt());
-    }
-
-    private String randomIntegerString() {
-        String s = Integer.toString(randomInt());
-        while (s.length() > 9) {
-            s = s.substring(1);
-        }
-
-        // Make sure the result is a valid Integer String:
-        return Integer.toString(Integer.parseInt(s));
-    }
-
-    private double randomDecimal() {
-        return random.nextDouble();
-    }
-
-    private String randomDecimalString() {
-        DecimalFormat decimalFormat = new DecimalFormat("####.#####");
-        return decimalFormat.format(randomDecimal());
-    }
-
-    //    private ActivityResult cancelledResult() {
-    //        return new ActivityResult(RESULT_CANCELED, null);
-    //    }
-    //
-    //    private ActivityResult okResult(@Nullable Intent data) {
-    //        return new ActivityResult(RESULT_OK, data);
-    //    }
-
-    private String randomIntegerSeparator() {
-        int number = 123456;
-        NumberFormat numberFormat = NumberFormat.getNumberInstance(Locale.US);
-        return numberFormat.format(number);
-     }
-
-    //endregion
-
     private static class AllWidgetsFormTestRule extends FormActivityTestRule {
```

Comment:
We probably don't need this class any more. The test should just use the `FormActivityTestRule` directly.